app/build.gradle

```diff
@@ -79,22 +79,23 @@ android {
 }
 
 ext {
-    androidxLibVersion = '1.0.0'
-    exoPlayerLibVersion = '2.10.8'
-    roomDbLibVersion = '2.1.0'
-    leakCanaryLibVersion = '1.5.4' //1.6.1
-    okHttpLibVersion = '3.12.6'
-    icepickLibVersion = '3.2.0'
-    stethoLibVersion = '1.5.0'
-    markwonVersion = '4.2.1'
-    checkstyleVersion = '8.31'
+    icepickVersion = '3.2.0'
+    checkstyleVersion = '8.32'
+    stethoVersion = '1.5.1'
+    leakCanaryVersion = '2.2'
+    exoPlayerVersion = '2.10.8'
+    androidxLegacyVersion = '1.0.0'
+    androidxLifecycleVersion = '2.2.0'
+    androidxRoomVersion = '2.2.5'
+    groupieVersion = '2.8.0'
+    markwonVersion = '4.3.1'
```

Comment:
Why did you change the order? And if we change the order, it should be for lexicographical order.

Reply:
I changed the order to be the same as down there. Down there I changed the order in a way that seems more logical to me.